packages/graphql/src/modules/user/presentation/controllers/facebookSessionController.spec.ts

```diff
@@ -0,0 +1,66 @@
+import * as http from 'http';
+
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import request from 'supertest';
+
+import app from '../../../../app';
+import {connectMongoose, clearDbAndRestartCounters, disconnectMongoose} from '../../../../shared/tests/helpers';
```

Comment:
what about having some aliases here? I think you can do that in babel

Reply:
Do you mean to avoid the ../../.. ...?